src/octoprint/users.py

```diff
@@ -216,10 +216,12 @@ def hasBeenCustomized(self):
 ##~~ FilebasedUserManager, takes available users from users.yaml file
 
 class FilebasedUserManager(UserManager):
-	def __init__(self):
+	def __init__(self, settingz=None):
 		UserManager.__init__(self)
-
-		userfile = settings().get(["accessControl", "userfile"])
+		if settingz is None:
```

Comment:
Instead of intentionally misspelling parameters, I'd rather suggest something like this:

``` python
import octoprint.settings

# [...]

def __init__(self, settings=None):
    UserManager.__init__(self)
    if settings is None:
        settings = octoprint.settings.settings()
    userfile = settings.get(["accessControl", "userfile"])
    # [...]
```